internal/provider/example_resource_test.go

```diff
@@ -40,10 +36,10 @@ import (
 //	})
 //}
 
-func testAccExampleResourceConfig(configurableAttribute string) string {
-	return fmt.Sprintf(`
-resource "netapp-ontap_example" "test" {
-  configurable_attribute = %[1]q
-}
-`, configurableAttribute)
-}
+//func testAccExampleResourceConfig(configurableAttribute string) string {
```

Comment:
comments needed?

Reply:
THe new linter will not allow us to have any function that has no use. THe example file have no use so I'm comment them all out.